mods/cnc/maps/nod07c/nod07c.lua

```diff
@@ -0,0 +1,186 @@
+	Reinforcements.Reinforce(player, NodUnitsRockets, { ReinforcementsBottomSpawn.Location, ReinforcementsRocketsRally.Location }, 10)
+	Trigger.AfterDelay(DateTime.Seconds(3), function()
+		Reinforcements.Reinforce(player, NodUnitsGunners, { ReinforcementsBottomSpawn.Location, ReinforcementsGunnersRally.Location }, 10)
+		Reinforcements.Reinforce(player, NodUnitsFlamers, { ReinforcementsTopSpawn.Location, ReinforcementsFlamersRally.Location }, 10)
+	end)
+	Trigger.AfterDelay(DateTime.Seconds(5), function()
+		local unitsA = Reinforcements.ReinforceWithTransport(player, 'tran.in', NodUnitsVehicles1, { GunboatRight.Location, ReinforcementsHelicopter1Rally.Location }, { GunboatRight.Location })[2]
+	
+		Trigger.AfterDelay(DateTime.Seconds(3), function()
+			local unitsB = Reinforcements.ReinforceWithTransport(player, 'tran.in', NodUnitsVehicles2, { GunboatRight.Location, ReinforcementsHelicopter2Rally.Location }, { GunboatRight.Location })[2]
+
+			Utils.Do(unitsB, function(unit)
+				unitsA[#unitsA + 1] = unit
+			end)
+
+			Trigger.OnAllKilled(unitsA, function()
+				if not defendersActive then
+					player.MarkFailedObjective(NodObjective3)
+				end
+			end)
+		end)
+	end)
+
+	Trigger.OnAllKilled(Engineers, function()
+		if not player.IsObjectiveCompleted(NodObjective1) then 
+			player.MarkFailedObjective(NodObjective1)
+		end
+	end)
+end
+
+SendGuards = function(team)
+	for type, amount in pairs(team.units) do
+		local count = 0
+		local actors = enemy.GetActorsByType(type)
+		Utils.Do(actors, function(actor)
+			if actor.IsIdle and count < amount then
+				actor.Patrol(team.waypoints, true, DateTime.Seconds(25))
+				count = count + 1
+			end
+		end)
+	end
+end
+
+SendDefenders = function(team)
+	defendersActive = true
+	player.MarkCompletedObjective(NodObjective3)
+	Trigger.AfterDelay(DateTime.Seconds(3), function()
+		for type, amount in pairs(team.units) do
+			local count = 0
+			local actors = enemy.GetActorsByType(type)
+			Utils.Do(actors, function(actor)
+				if actor.IsIdle and count < amount then
+					actor.AttackMove(waypoint0.Location)
+					count = count + 1
+				end
+			end)
+		end
+	end)
+end
+
+Utils.Do(GDIEntrance, function(actor)
+	Trigger.OnDamaged(actor, function()
+		if not defendersActive then
+			SendDefenders(GDI2)
+		end
+	end)
+end)
+
+Trigger.OnKilled(Gunboat, function()
+	GunboatCamera.Destroy()
+end)
+
+Trigger.OnKilled(GDIHpad, function()
+	if not player.IsObjectiveCompleted(NodObjective1) then
+		player.MarkFailedObjective(NodObjective1)
+	end
+end)
+
+Trigger.OnDamaged(GDIBuilding11, function()
+	SendGuards(MTANK)
+end)
+
+Utils.Do(Targets, function(actor)
+	Trigger.OnKilled(actor, function()
+		targetsKilled = targetsKilled + 1
+	end)
+end)
+
+Trigger.OnCapture(GDIHpad, function()
+	hpadCaptured = true
+	player.MarkCompletedObjective(NodObjective1)
+	local Orca = Actor.Create("orca", true, { Owner = player, Location = GDIHpad.Location })
```

Comment:
I was just thinking about to preplace an orca which is owned by a neutral faction... but then I realized that I can't place it at the center of the hpad... so, no Idea how to do it.